hq_superset/oauth.py

```diff
@@ -10,3 +12,33 @@ def oauth_user_info(self, provider, response=None):
             user = self.appbuilder.sm.oauth_remotes[provider].get("api/v0.5/identity/", token=response).json()
             logging.debug("user - {}".format(user))
             return user
+
+    def create_custom_role_perms(self, domains):
+        for domain_name in domains.items():
+            role_created = self.create_custom_role(domain_name)
+            if not role_created:
+                continue
+            self.create_schema(domain_name)
+            pv = self.create_custom_permission(domain_name)
+            sesh = self.get_session
+            role_created.permissions = pv
+            sesh.merge(role_created)
+            sesh.commit()
+
+    def create_custom_permission(self, domain_name):
+        schema_access_perm_name = "schema_access"
+        schema_access_view_menu_name = "[{}].[{}]".format("superset", domain_name)
```

Comment:
What is this permission for?

Reply:
Format for schema access permission is <db>.<schema name>